test/e2e/kubectl/kubectl.go

```diff
@@ -722,7 +722,7 @@ metadata:
 			framework.ExpectError(err)
 			gomega.Expect(err).To(gomega.ContainSubstring("Using in-cluster namespace"))
 			gomega.Expect(err).To(gomega.ContainSubstring("Using in-cluster configuration"))
-			gomega.Expect(err).To(gomega.ContainSubstring("Authorization: Bearer invalid"))
+			gomega.Expect(err).To(gomega.ContainSubstring("Authorization: Bearer <masked>"))
```

Comment:
go ahead and drop this entire line, it isn't checking anything useful now (previously it was checking our overridden token was used)

Reply:
@liggitt 
This line has been removed.